web/html/core-html-elements/update-images.md

```diff
@@ -90,12 +90,12 @@ If an image does not have an assigned height and width using HTML or CSS, how wi
 
 ???
 
-* `img src="image.jpg" alt="Image Description"`
+* `<img src="image.jpg" alt="Image Description">`
 * `alt`
 * `.img`
 * At its original image size.
-* `image src="image.jpg" alt="Image Description"`
-* `img alt="image.jpg" src="Image Description"`
+* `<image src="image.jpg" alt="Image Description">`
+* `<img alt="image.jpg" src="Image Description">`
```

Comment:
no need for backticks as the answers already display as if enclosed by backticks
![image](https://user-images.githubusercontent.com/19802557/77753526-95579c80-7021-11ea-8efc-169b65754187.png)

```suggestion
* <image src="image.jpg" alt="Image Description">
* <img alt="image.jpg" src="Image Description">
```